### ISSUE TITLE
Execution takes longer on each re-run

### ISSUE THREAD

**Issue by zewa666** [aurelia/benchmarks#1]
Hi,

I've played around with the benchmarking tool and it seems if you execute the test multiple times (re run with the run-button) it takes longer and longer.

Also the Chrome profiles tells us that memory consumption is continuously increasing. Is there maybe a bug with the Benchmarking Lib?
![2015-03-19_12h03_48](https://cloud.githubusercontent.com/assets/2861414/6729557/d0dca72c-ce33-11e4-9e57-941a76794576.png)

**Comment by zewa666** [comment]
Also I thought it's maybe because of using setTimeout, but another test I've created, like this one

``` javascript
import {ObserverLocator} from 'aurelia-framework';
var observer = new ObserverLocator();

export default (deferred) => {
  var observedObj = {
    observedProp: 'Test'
  };

  observer.getObserver(observedObj, 'observedProp')
    .subscribe(() => {
      deferred.resolve();
    });

  observedObj.observedProp = 'Test1';

}
```

also behaved the same way.

**Comment by zewa666** [comment]
Ok I've tried to remove the repeater from the UI and only show an Interpolation like here:

```
<div>test ${microTests[0].elapsed}</div>
<button click.trigger="runTests()" class="btn btn-default">Run</button>
```

now the timeline tells me something like this. I'm no expert about profiling but that looks somehow better. So I guess the repeater seems to be leaking memory somehow

![2015-03-19_12h41_04](https://cloud.githubusercontent.com/assets/2861414/6729713/937b0386-ce35-11e4-82a0-33ebeea35b0d.png)

**Comment by OdeToCode** [comment]
Interesting - I'll check on this. I am in the process of rearranging the execution schedule, so I'll see if that has an impact, too. Currently the tests might try to run in parallel, which isn't what we want, but that will change shortly.

**Comment by jdanyow** [comment]
@OdeToCode  nice- I was going to ask you about the parallel execution thing- wasn't sure if benchmark.js was taking care of executing them one at a time or not.

**Comment by zewa666** [comment]
Ok so with the refactoring of executing the tests one after another, it seems that test results have stabilized a bit, values now going up and down with multiple re-tries. Also memory consumption looks ... better?

![2015-03-23_17h38_55](https://cloud.githubusercontent.com/assets/2861414/6785094/ee7693a8-d183-11e4-9060-9b59fc5e2188.png)

Nevertheless I think we'd still have to take a closer look to the repeater, but as far as goes with the Benchmark repo I think I can close this issue for now.
